### PR TITLE
gentx validation github action

### PR DESCRIPTION
**Pull request by alexanderbez**

### CODE REVIEW

File: .github/workflows/validate.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v2.1.5
+        with:
+          go-version: 1.19
+      - uses: actions/checkout@v2
+      - uses: technote-space/get-diff-action@v6.0.1
+        with:
+          PATTERNS: |
+            gentxs/*.json
```
Comment (alexanderbez) [inline .github/workflows/validate.yml:27]: Change this if you expect gentxs to be somewhere else

File: gentx-validator/main.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/signing"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/palomachain/paloma/app"
+)
+
+var (
+	minCommissionAllowed = sdk.MustNewDecFromStr("0.02")
```
Comment (alexanderbez) [inline gentx-validator/main.go:20]: If you don't want a min commission, you can also remove this too!

Comment (verabehr) [reply gentx-validator/main.go:20]: @taariq thoughts? I think we can remove this

Comment (taariq) [reply gentx-validator/main.go:20]: we want a minimum commission so that we do not encourage race to bottom competition. I recommend we hold to 2%.

**Comment by verabehr** [comment]
looks good to me. Updated the gentx location as we don't have a specific folder for these atm

**Review by verabehr — APPROVED** [review]

**Pull request merged**